internal/pkg/service/appsproxy/dataapps/appconfig/appconfig_test.go

```diff
@@ -223,7 +223,6 @@ func TestLoader_LoadConfig(t *testing.T) {
 	}
 
 	for _, tc := range testCases {
-		tc := tc
 		t.Run(tc.name, func(t *testing.T) {
```

Comment:
I already removed it in API PRs, so please rebase and merge it, after the API part will be merged, at first.